{"name": "test_checklogin[test_register2]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 1 == 0\n  -1\n  +0", "trace": "request = <SubRequest 'test_register' for <Function test_checklogin[test_register2]>>\n\n    @pytest.fixture(params=excelinfo)\n    def test_register(request):\n        ss=request.param\n        url=ss[0]\n        register_service=ss[1]\n        login_service=ss[2]\n        username=ss[3]\n        password=ss[4]\n        payload = {'service':ss[1],'username':ss[3],'password':ss[4],'is_allow_many':\"1\",'app_key':\"784DBD349CE644ABD3684A562C159585\"}\n        login=src.test.common.testrequest.TestRequest()\n        s=login.checkmethord(\"get\",ss[0],payload)\n        err_code=s[\"data\"][\"err_code\"]\n        # uuid=s[\"data\"][\"uuid\"]\n        # token=s[\"data\"][\"token\"]\n        print(\"s=======\",s)\n>       assert err_code  ==0\nE       assert 1 == 0\nE         -1\nE         +0\n\ntest_regandlogin.py:29: AssertionError"}, "parameters": [{"name": "test_register", "value": "['http://hn216.api.yesapi.cn/', 'App.User.Register', 'App.User.Login', 'autotest10', '174b086fc6358db6154bd951a8947837']"}], "start": 1572601012732, "stop": 1572601012732, "uuid": "8a4c5c25-f297-4317-aaea-498006562a90", "historyId": "c000e0f6f58166922217bb5f8f17c965", "testCaseId": "e5b734ac259aa4ca5f57d383614687f1", "fullName": "test_regandlogin#test_checklogin", "labels": [{"name": "suite", "value": "test_regandlogin"}, {"name": "host", "value": "DESKTOP-4A0JR1L"}, {"name": "thread", "value": "1808-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_regandlogin"}]}